src/engine/Actor.ts

```diff
@@ -484,14 +486,34 @@ export class ActorImpl extends Class implements IActionable, IEvented {
       }
    }
 
+   private _capturePointerEvents: string[] = [
```

Comment:
This is good; one thing we could do is increase static checking consistency by typing this to a string enum or string union (e.g. `PointerEvents`)

Reply:
I think most of the event strings in Events.ts could benefit from becoming Enums over types. Perhaps a proper refactor should be added as an issue?

@kamranayub @eonarheim fyi

Reply:
@pathurs I agree, doing a proper refactor is probably out of scope for this PR. Let's open a new issue to look at consolidating Event types as enums. At the very least we could still do a string type union something like? `type PointerEvents = 'pointerup' | 'pointerdown' | 'pointermove' ...`

Reply:
@pathurs I missed this before in my summary, could you also update these string arrays to use a type union on lines 494 & 489? Last request I promise